service/test/y2storage/agama_proposal_test.rb

```diff
@@ -63,18 +63,17 @@ def partition_config(name: nil, filesystem: nil, size: nil)
   config = Agama::Storage::Configs::Partition.new
   block_device_config(config, name: name, filesystem: filesystem)
 
-  if size
-    config.size = Agama::Storage::Configs::Size.new.tap do |size_config|
-      size_config.min = size
-      size_config.max = size
-    end
+  config.size = Agama::Storage::Configs::Size.new.tap do |size_config|
```

Comment:
This will not be needed thanks to the solvers introduced by https://github.com/openSUSE/agama/pull/1617. BTW, in general I am trying to generate configs from JSON hashes. I think it makes the tests easier to read and maintain.